cnd/Cargo.toml

```
@@ -33,6 +33,7 @@ hyper = "0.12"
 lazy_static = "1"
 libp2p = { version = "0.12" }
 libp2p-comit = { path = "../libp2p-comit" }
+libsqlite3-sys = { version = ">=0.8.0, <0.13.0", features = ["bundled"] } # bundling sqlite removes the dependency on the shared library
```

```suggestion
libsqlite3-sys = { version = ">=0.8.0, <0.13.0", features = ["bundled"] }
```
People should RTFM if they wonder what it is.

---

True, the commit message should be good enough in this case :)